packages/cardhost/app/controllers/cards/view.js

```diff
@@ -29,4 +30,15 @@ export default class ViewCardController extends Controller {
     this.themerOptions.push({name: 'Custom theme'})
     this.router.transitionTo('cards.view', this.model, { queryParams:{ editingCss: true } })
   }
+
+  @action
+  save() {
+    this.model.save()
```

Comment:
There was a "Save" button on the layout page, but it didn't do anything, so now we add that functionality.